ticketutil/bugzilla.py

```diff
@@ -149,10 +207,12 @@ def _create_ticket_request(self, params):
         """
         # Attempt to create ticket.
         try:
-            if self.token:
-                params['token'] = self.token
+            #if self.token:
+            #    params['token'] = self.token
+            params.update(self.auth.getCredentials())
+            
 
-            r = self.s.post(self.rest_url, json=params)
+            r = self.s.post(self.rest_url, json=params, verify=False)
```

Comment:
modified code for "s.verify = False" in _create_requests_session() and removed the remaining entries.